src/style.scss

```diff
@@ -26,6 +31,189 @@ ul {
     height: 100%;
 }
 
+/* MODAL */
+/* Background effect. It covers the entire site */
+.modal {
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Overlay effect: translucent background: black w/ partial opacity */
+    z-index: 1; /* Overlay effect: positioned over other containers */
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    position: fixed; /* Fix position on the top-left corner*/
+    top: 0;
+    left: 0;
+    overflow: auto; /* Enable scroll if needed */
+    padding-top: 80px; /* Location of the content container */
+}
+
+#navBar {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #000000;
+  text-align: right;
+}
+
```

Comment:
can remove these (.modal and #navBar). These are probably from a merge conflict right? I moved .modal into ResponsePageStyles (since that's where the modal is called) and the #navBar into HomepageStyles 

Unless you wrote these to use for something else!